fiftyone/multimodal/ingest/__init__.py

```
@@ -5,3 +5,40 @@
+    Returns:
+        a list of :class:`fiftyone.multimodal.SceneInventory` instances
+    """
+    inventories = []
+
+    for filepath in filepaths:
+        if storage.exists(filepath) and adapter.can_read(filepath):
+            inventories.append(adapter.get_scene_inventory(filepath))
+            continue
+
+        if storage.isdir(filepath):
+            for filepath in storage.list_files(
```

Ruff (0.15.12)

[warning] Loop control variable `filepath` overrides iterable it iterates

(B020)